packages/core/src/DatadogProvider/Buffer/BoundedBuffer.ts

```diff
@@ -154,4 +171,33 @@ export class BoundedBuffer extends DatadogBuffer {
 
         throw new Error('Could not generate random Buffer id');
     };
+
+    private addTelemetryEvent = (
+        message: string,
+        stack: string,
+        kind: string
+    ) => {
+        if (this.telemetryBuffer[kind]) {
+            this.telemetryBuffer[kind].occurences++;
+        } else {
+            this.telemetryBuffer[kind] = {
+                message,
+                stack,
```

Comment:
but I guess the errors of the same kind may have different stack?

Reply:
Yes, but I think it's more important to group errors to understand its impact and prevent sending too many events to telemetry as I don't think the part of the stack that might change is very important to us here (and we don't have the sourcemaps to deobfuscate it).

Reply:
indeed, by adding `stack` we increase cardinality. It is ok for me to have *some* stack as long as it allows to address the real issue.